ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -178,6 +178,14 @@ private[testtool] final class ParticipantTestContext private[participant] (
       .uploadDarFile(new UploadDarFileRequest(bytes))
       .map(_ => ())
 
+  def uploadDarRequest(bytes: ByteString): UploadDarFileRequest =
+    new UploadDarFileRequest(bytes, nextSubmissionId())
+
+  def uploadDarFile(request: UploadDarFileRequest): Future[Unit] =
```

Comment:
Can you change `def uploadDarFile(ByteString)` to delegate to this one please?

Reply:
Done